### DISCUSSION TITLE
Image reader rotating images 90 degrees to the left

### DISCUSSION THREAD

**Discussion by NMVRodrigues** [Project-MONAI/MONAI#1819] [Q&A]
Using the provided `monai.transforms.LoadImage` to load Nifti images rotates them 90 degrees to the left.
Is the rotation intended?
Is there any way to overcome this issue and load them without the rotation?

Here is the code showing a slice of the image:

```
from monai.transforms import LoadNifti 
loader = LoadNifti(image_only=True) 
img_array = loader(images[0]) 
plt.figure() 
plt.imshow(img_array[:, :, 0], cmap="gray") 
plt.show()
```

Here is the output image:

![image](https://user-images.githubusercontent.com/26222830/111996541-09802f80-8b12-11eb-9255-df6ffa8f7501.png)

And here is the correct image orientation as seen on ITK Snap:

![image](https://user-images.githubusercontent.com/26222830/111996615-1e5cc300-8b12-11eb-9976-df47c0147ae7.png)

EDIT:

The slices are not the same but they are from the same volume.

**Comment by wyli** [comment]
perhaps you can specify `origin='lower' or origin='upper'` options in `imshow`? I'm converting this into a discussion

**Reply by NMVRodrigues** [reply]
So the problem lies within the way matplotlib represents the data, and not how it was loaded?

**Reply by wyli** [reply]
I wouldn't say it's a problem, they are different conventions for visualisation...

**Reply by NMVRodrigues** [reply]
Alright yes that is fair, not necessarily a problem.
Although I'm still unsure the behaviour is caused by matplotlib, since saving the image using pythons png package also saves with the rotation, so I'm more inclined to the hypothesis that it's the Nifti Loader / LoadImage method that is rotating the matrix.

**Reply by rijobro** [reply]
ITKSnap will rotate your image to align the LPS of the patient with the viewer. In this way, whenever you load an image, the patient will always be oriented in the same direction (provided the metadata is correct). However, MONAI's readers (ITK, nibabel, etc.) are just reading in the array. There's therefore no guarantee that an image displayed in ITKSnap should be displayed in the same orientation as the raw array as read by one of MONAI's readers.

If you want your images to be reorientated so that they look the same as in ITKSnap, I imagine you could do this using our `Orientation` transform: https://github.com/Project-MONAI/MONAI/blob/master/monai/transforms/spatial/array.py#L205.

**Reply by NMVRodrigues** [reply]
Alright makes much more sense now, thanks for the clarification!
